src/util/indent/lexi.c

```diff
@@ -335,6 +337,10 @@ lexi()
 		  break;
 	      }
 	    buf_ptr++;
+	    if (lpc && qchar == '\'' && !(isalnum(*buf_ptr)||*buf_ptr=='_')) {
```

Comment:
Could you please add a comment, what this condition checks for?
Also pay attention that buf_ptr might point after the end of the buffer (the condition is between incrementation of the pointer and the check for the end of the buffer), so dereferencing might not be a good idea.